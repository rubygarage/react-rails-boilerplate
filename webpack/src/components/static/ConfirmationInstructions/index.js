import React from 'react';
import { FormattedMessage } from 'react-intl';

function ConfirmationInstructions() {
  return (
    <main className="container">
      <div className="jumbotron">
        <h1 className="display-3"> <FormattedMessage id="confirmation_instructions.thanks" /> </h1>
        <p className="lead"> <FormattedMessage id="confirmation_instructions.check_email" /> </p>

        <ul className="list-unstyled">
          <li> <FormattedMessage id="confirmation_instructions.help_header" /> </li>

          <ul>
            <li> <FormattedMessage id="confirmation_instructions.help_spam" /> </li>
            <li> <FormattedMessage id="confirmation_instructions.help_email" /> </li>
            <li> <FormattedMessage id="confirmation_instructions.help_support" /> </li>
          </ul>
        </ul>
      </div>
    </main>
  );
}

export default ConfirmationInstructions;
