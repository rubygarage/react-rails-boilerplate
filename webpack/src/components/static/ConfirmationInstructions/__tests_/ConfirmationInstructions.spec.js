import React from 'react';
import { shallow } from 'enzyme';
import ConfirmationInstructions from 'components/static/ConfirmationInstructions';

describe('<ConfirmationInstructions />', () => {
  it('renders ConfirmationInstructions component', () => {
    const component = shallow(<ConfirmationInstructions />);

    expect(component).toMatchSnapshot();
  });
});
