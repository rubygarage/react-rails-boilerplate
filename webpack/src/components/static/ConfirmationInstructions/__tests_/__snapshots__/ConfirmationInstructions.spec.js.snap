// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConfirmationInstructions /> renders ConfirmationInstructions component 1`] = `
<main
  className="container"
>
  <div
    className="jumbotron"
  >
    <h1
      className="display-3"
    >
       
      <FormattedMessage
        id="confirmation_instructions.thanks"
        values={Object {}}
      />
       
    </h1>
    <p
      className="lead"
    >
       
      <FormattedMessage
        id="confirmation_instructions.check_email"
        values={Object {}}
      />
       
    </p>
    <ul
      className="list-unstyled"
    >
      <li>
         
        <FormattedMessage
          id="confirmation_instructions.help_header"
          values={Object {}}
        />
         
      </li>
      <ul>
        <li>
           
          <FormattedMessage
            id="confirmation_instructions.help_spam"
            values={Object {}}
          />
           
        </li>
        <li>
           
          <FormattedMessage
            id="confirmation_instructions.help_email"
            values={Object {}}
          />
           
        </li>
        <li>
           
          <FormattedMessage
            id="confirmation_instructions.help_support"
            values={Object {}}
          />
           
        </li>
      </ul>
    </ul>
  </div>
</main>
`;
